src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -592,8 +596,8 @@ def performAggregateInterferenceCheck(self):
         if self.num_peer_sases > 0:
           ppa_ap_iap_ref_values = self.ppa_ap_iap_ref_values_list[index]
         else:
-          ppa_ap_iap_ref_values = [interference.dbToLinear(iap.THRESH_PPA_DBM_PER_RBW)] * len(ppa_aggr_interference)
-        # Compare the interference values calculated from both models.
+          ppa_ap_iap_ref_values = interference.dbToLinear(iap.THRESH_PPA_DBM_PER_RBW)
```

Comment:
why are we removing multiplying "len(ppa_aggr_interference)" ? Also same question to the other places that the multiplying parts are removed.

Reply:
When there are no peer SASs the interference to compare against is a single scalar threshold. There is no need to construct a list of the scalar, this is supported by the changes on lines 697-711 which compare directly against the number when it is a number.

Reply:
ok, thanks.